Makefile

```diff
@@ -1308,6 +1308,23 @@ partialclean::
 
 beforedepend:: bytecomp/opcodes.ml
 
+# Testing the parser -- see parsing/HACKING.adoc
+
+SOURCE_ML_FILES=$(shell git ls-files | grep '\.ml$$')
```

Comment:
can replace this by `$(shell git ls-files *.ml)`

Reply:
I tried `git ls-files *.ml` in my terminal and it doesn't work, but now that you say it `git ls-files '*.ml'` does work -- I guess you have to present the shell from globbing the argument first. I will change this, thanks!